app/controllers/events_controller.rb

```diff
@@ -1,3 +1,45 @@
 class EventsController  < ApplicationController
 
+	def new
+		@event = Event.new
+	end
+
+
+
+	def create
+ 		@event = Event.new(event_params)
+ 
+  		@event.save
+  		redirect_to @event
+	end
+
+	def edit
+    @event = Event.find(params[:id])
+  end
+
+  def update
+    @event = Event.find(params[:id])
+    @event.update(event_params)
+
+    redirect_to event_path(@event)
+  end
+
+
+	def destroy
+		@event = Event.find(params[:id])
+		@event.destroy
+
```

Comment:
I would get rid of the extra blank line 31 as well

Reply:
Fixed with rubocop e74418477e56ceb387e855a1c9b90138b9f90713